src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -119,11 +120,12 @@ def __init__(
         if "target" not in self.target_fields:
             raise KeyError('"target" is expected to be in the fields to subclass but it was not included')
 
+        self.REQUIRED_ANNOTATION_FIELDS = {"target", "img_path", "rescaled_img_shape"}
```

Comment:
It's a because all the other attributes are variables (at least they depend on the init params), and this one is the only constant, so I used the convention for constants to emphasize on it.
But I don't mind, we can just make it private, I guess it is good as well

Reply:
changed in 1205d4148559afc71221c13d55e4c940ec4a1c54